### PR TITLE
Add book subtitle to lists

### LINKED ISSUE #4783: Add book subtitles to lists
Problem: 
right now, putting subtitles into the subtitle box doesn't work out for titles that have the same name, as they all look the same in lists - due to the subtitles not showing up in lists. This forces people to write what should be in the subtitles in the titles and is just a mess and overall hard to read and disorganized.

Solution:
Add subtitles to lists, so that subtitles can be written where they should and everything's more readable.

Example:
image

Would look like all of them had California Treasures as the title if the part after it - the grades and books were in the subtitles instead. Adding subtitles would differentiate them in the lists and there's enough space to do so too.

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #4783 

Added subtitle to items(books) present in lists
### Screenshot

image

### Stakeholders

@BrittanyBunk @jdlrobson

### CODE REVIEW

**Comment by BrittanyBunk** [comment]
@Yashs911 yay - thank you so much!

**Comment by BrittanyBunk** [comment]
@Yashs911 I just tried it on https://openlibrary.org/people/ghoulgalpals/lists/OL194586L/california_treasures . It's not working. Any reason?

**Comment by Yashs911** [comment]
> @Yashs911 I just tried it on https://openlibrary.org/people/ghoulgalpals/lists/OL194586L/california_treasures . It's not working. Any reason?

@BrittanyBunk First this PR needs to be merged and after a deployment, you will be able to see the subtitles

File: static/css/base/headings.less [hunk 1]
```diff
@@ -106,6 +106,13 @@ h4 {
     font-family: @lucida_sans_serif-1;
     font-weight: normal;
   }
+  &.work-subtitle {
```
Comment (jdlrobson) [inline static/css/base/headings.less:109]: The base/headings.less file is meant to define common heading styles. This particular style is however not generic- it's part of the lists component.

Please move to static/css/components/mybooks-list.less

Comment (Yashs911) [reply static/css/base/headings.less]: Done.

File: openlibrary/templates/type/list/view.html [hunk 2]
```diff
@@ -206,6 +206,8 @@ <h1>$list.name</h1>
                         <h3 class="booktitle">
                             <a href="$seed.url" class="results">$seed.title</a>
                         </h3>
+                        $if seed.type in ['edition', 'work'] and doc.subtitle:
+                            <h4 class="work-subtitle">$doc.subtitle</h4>
```
Comment (jdlrobson) [inline openlibrary/templates/type/list/view.html:210]: Am not 100% convinced h4 is appropriate is here, but also not a blocker.

**Review by jdlrobson — APPROVED** [review]
over to @mekarpeles to confirm from product pov

File: openlibrary/templates/type/list/view.html (hunk 2 above)
Comment (mekarpeles) [reply openlibrary/templates/type/list/view.html:210]: @bpmcneilly may have comments about the document structure. Given the title is currently a h3, not opposed to merging as is. Seems to add useful data to the document / display. Happy for there to be a followup issue opened if a different semantic structure is useful.

File: static/css/components/mybooks-list.less [hunk 3]
```diff
@@ -26,6 +26,14 @@
     }
   }
 
+  h4.work-subtitle {
```
Comment (mekarpeles) [inline static/css/components/mybooks-list.less:29]: would have been great if we could have gotten away w/ not adding additional css! Keep it simple, re-use what we can :)

**Review by mekarpeles — APPROVED** [review]

**Comment by mekarpeles** [comment]
Thank you @jdlrobson for the review and the tag. I think this looks good enough to go.

File: static/css/components/mybooks-list.less (hunk 3 above)
Comment (Yashs911) [reply static/css/components/mybooks-list.less:29]: If we plan to restructure the semantic I will remove the additional CSS.

**Pull request merged**